django/contrib/admin/options.py

```diff
@@ -829,6 +829,15 @@ def get_action(self, action):
             description = capfirst(action.replace('_', ' '))
         return func, action, description
 
+    def get_empty_value_display(self):
```

Comment:
Do you mean like this:

    def birthday_display(self):
        return self.brithday

    birthday_display.empty_value_display = '-unknown-'